cdap-distributions/src/COPYRIGHT/io.cdap.plugin:format-orc:2.8.0-SNAPSHOT/COPYRIGHT

```diff
@@ -0,0 +1,232 @@
+Copyright © 2014-2015 Cask Data, Inc.
```

Comment:
Do we need to put our own licenses here?

Reply:
see :  https://github.com/cdapio/cdap/pull/13722#discussion_r738600982 :)

Reply:
IMHO it's weird, but who am I to question traditions :)